viewer/packages/cad-styling/src/prioritized/mergeboxes/SmartMergeBoxes.ts

```diff
@@ -56,16 +70,20 @@ export class SmartMergeBoxes implements BoxClusterer {
     this.addedSinceSquash += 1;
   }
 
+  get boxCount(): number {
+    return this.resultBoxes.length;
+  }
+
   addBoxes(boxes: Iterable<Box3>): void {
     for (const box of boxes) {
-      this.addBox(box);
+      this.addBox(box.expandByScalar(0.3));
```

Comment:
Why expand? Also, this ends up modifying the input box which can be problematic I guess.

Reply:
Ah, the expand could probably be removed. It was there to ensure nearby small objects were clustered together, but I think the current heuristic works well for that anyway

And it should have cloned the box first :sweat_smile: